runtime/makelib/mkconstants.mk.ftl

```diff
@@ -25,13 +25,17 @@
 
 # Define the Java Version we are compiling
 ifndef VERSION_MAJOR
+<#if uma.spec.properties.JAVA_SPEC_VERSION.defined>
+VERSION_MAJOR := ${uma.spec.properties.JAVA_SPEC_VERSION.value}
+<#else>
 $(error VERSION_MAJOR is not set from extensions code)
+</#if>
 endif
 export VERSION_MAJOR
 
 # Define full Java Version
 ifndef OPENJDK_VERSION_NUMBER_FOUR_POSITIONS
-$(error OPENJDK_VERSION_NUMBER_FOUR_POSITIONS is not set from extensions code)
+OPENJDK_VERSION_NUMBER_FOUR_POSITIONS := $(VERSION_MAJOR).0.0.0
```

Comment:
I don't like to hard code the value instead of giving an error, in case the value stops being set properly in the future because then the hard coded value would be incorrect in a production build. JDK11 sets OPENJDK_VERSION_NUMBER_FOUR_POSITIONS to `$VERSION_FEATURE.$VERSION_INTERIM.$VERSION_UPDATE.$VERSION_PATCH`. can we get all of these variables to create the proper version info?

Reply:
OPENJDK_VERSION_NUMBER_FOUR_POSITIONS was never needed here: it has been removed.